apps/web-giddh/src/app/ledger/ledger.component.ts

```diff
@@ -3487,6 +3465,15 @@ export class LedgerComponent implements OnInit, OnDestroy {
         this.router.navigate(['pages', 'settings', 'integration', 'payment']);
     }
     
+    /**
+     * Close All Account Dropdown
+     *
+     * @memberof LedgerComponent
+     */
+    public closeAllAccountDropdown(): void {
+        this.dropDowns.forEach((alertInstance, i) => alertInstance?.closeDropdownPanel());
```

Comment:
```suggestion
        this.dropdowns.forEach((alertInstance, i) => alertInstance?.closeDropdownPanel());
```
Please correct name for dropdowns